coi/src/config.rs

```diff
@@ -30,8 +30,7 @@ use crate::{
 pub struct Config {
     shift_factor: f32,
     threshold: f32,
-    min_positive_cois: usize,
-    min_negative_cois: usize,
+    min_cois: usize,
```

Comment:
the corresponding env vars have never been set for any tenant so far, hence we can change the names without issues.